static/src/javascripts/projects/commercial/modules/dfp/define-slot.js

```diff
@@ -100,6 +113,11 @@ const defineSlot = (adSlotNode: Element, sizes: Object): Object => {
         slot.setTargeting('slot-fabric', fabricKeyValues.get(slotTarget));
     }
 
+    if (config.switches.adomik) {
+        slot.setTargeting('ad_group', adomikClassify());
+        slot.setTargeting('ad_h', new Date().getUTCHours().toString());
+    }
```

Comment:
Why does this need to be done at the slot level and not the page level?  It looks as though the targeting will be the same for all the slots on the page.